src/Features/Core/Portable/Completion/CompletionItemFlags.cs

```diff
@@ -29,12 +30,14 @@ internal enum CompletionItemFlags
         CachedAndExpanded = Cached | Expanded,
     }
 
-    internal static class CompletionItemFlagsExtensions
+    internal static class CompletionItemFlagsHelper
     {
-        public static bool IsCached(this CompletionItemFlags flags)
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
❔ Do we have an indication that this is not currently getting inlined?

Reply:
reverted